lib/matplotlib/colorbar.py

```diff
@@ -1164,6 +1191,39 @@ def remove(self):
             # use_gridspec was True
             ax.set_subplotspec(subplotspec)
 
+    def minorticks_on(self):
+        """
+        Turns on the minor ticks on the colorbar without extruding
+        into the "extend regions".
+        """
+        try:
+            ax = self.ax
+        except AttributeError:
+            return
+
+        if self.orientation == 'vertical':
+            long_axis = ax.yaxis
+        else:
+            long_axis = ax.xaxis
+
+        long_axis.set_minor_locator(_ColorbarAutoMinorLocator(self))
```

Comment:
Sorry to review in bits and drabs... 

What happens if the colorbar is logarithmic?  Is this function still a good idea?  Or should you be setting a `_ColorbarLogMinorLocator`?

Reply:
you are right... I completely overlooked that case. Will fix it.

Reply:
hmm... this is an interesting journey! From what I understand, LogLocator only returns Major ticks... I guess I will have to dig deeper.

Reply:
I think the log locator looks after the minor ticks, but I could be misremembering.  So in this case, you probably just need to be careful that "minorticks_on" just doesn't do anything so as not to step on the log locator's toes.